### ISSUE TITLE
some question about loss

### ISSUE THREAD

**Issue by bayminbx** [willylulu/RelGAN#3]
hi, thanks for you share your work. However, I want to ask Is there a bug in this line of code:

self.di_loss = K.square(K.minimum(inter_seed_rep, K.ones_like(inter_seed_rep) - inter_seed_rep) * K.ones_like(di_inter) - di_inter) + K.square(di_temp)

the operation * should be conducted the first dim?
